Code/GraphMol/SmilesParse/SmilesParse.h

```diff
@@ -32,11 +104,46 @@ struct RDKIT_SMILESPARSE_EXPORT SmilesParserParams {
   bool skipCleanup =
       false; /**<  skip the final cleanup stage (for internal use) */
 };
-RDKIT_SMILESPARSE_EXPORT RWMol *SmilesToMol(const std::string &smi,
-                                            const SmilesParserParams &params);
 
-RDKIT_SMILESPARSE_EXPORT Atom *SmilesToAtom(const std::string &smi);
-RDKIT_SMILESPARSE_EXPORT Bond *SmilesToBond(const std::string &smi);
+struct RDKIT_SMILESPARSE_EXPORT SmartsParserParams {
+  int debugParse = 0; /**< enable debugging in the SMARTS parser*/
+  std::map<std::string, std::string> *replacements =
+      nullptr;               /**< allows SMARTS "macros" */
+  bool allowCXSMILES = true; /**< recognize and parse CXSMILES extensions */
+  bool strictCXSMILES =
+      true; /**< throw an exception if the CXSMILES parsing fails */
+  bool parseName = true; /**< parse (and set) the molecule name as well */
+  bool mergeHs =
+      false; /**< toggles merging H atoms in the SMARTS into neighboring atoms*/
+  bool skipCleanup =
```

Comment:
If this is really "for internal use only", maybe it could be private and the reaction SMARTS parser could be a friend

Reply:
Again, we can't completely remove it without breaking backwards compatibility in v1, but I will at least get rid of the "internal use" note, which isn't helpful